### ISSUE TITLE
NUnit 3.6.0 don't find tests on .Net Standard 1.6 projects

### ISSUE THREAD

**Issue by galvesribeiro** [nunit/nunit#1981]
Visual Studio is throwing this:

```
------ Discover test started ------
NUnit Adapter 3.6.0.0: Test discovery starting
Error: Unable to get runner for this assembly. Check installation, including any extensions.
FileNotFoundException: Could not load file or assembly 'nunit.framework' or one of its dependencies. The system cannot find the file specified.
Dependent Assembly nunit.framework of C:\Users\gutem\documents\visual studio 2017\Projects\ClassLibrary1\ClassLibrary1\bin\Debug\netstandard1.6\ClassLibrary1.dll not found. Can be ignored if not a NUnit project.
NUnit Adapter 3.6.0.0: Test discovery complete
========== Discover test finished: 0 found (0:00:00.655) ==========
```

This is the sample .csproj:

```
<Project Sdk="Microsoft.NET.Sdk" ToolsVersion="15.0">
  <PropertyGroup>
    <TargetFramework>netstandard1.6</TargetFramework>
  </PropertyGroup>
  <ItemGroup>
    <Compile Include="**\*.cs" />
    <EmbeddedResource Include="**\*.resx" />
  </ItemGroup>
  <ItemGroup>
    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.0.0-preview-20170113-02" />
    <PackageReference Include="NETStandard.Library" Version="1.6.1" />
    <PackageReference Include="NUnit" Version="3.6.0" />
    <PackageReference Include="NUnit3TestAdapter" Version="3.6.0" />
  </ItemGroup>
</Project>
```
With this sample test:

```
[TestFixture]
    public class Class1
    {
        string teststring = "";

        [SetUp]
        public void SetUp()
        {
            teststring = "AAA";
        }

        [Test]
        public void TestDemo()
        {
            Assert.True(teststring.Equals("AAA"));
        }
    }
```

Do we need to do anything else?

Thank you.

**Comment by jnm2** [comment]
@CharliePoole I wonder if this is https://github.com/nunit/nunit3-vs-adapter/issues/296?

@galvesribeiro Can you see if the NUnit console runner runs the tests properly?

**Comment by galvesribeiro** [comment]
@jnm2 what should I do to use the console runner on .net standard projects?

**Comment by JustinRChou** [comment]
Seems like it.
NUnit Adapter 3.6.1.0 has the same issue.

**Comment by jnm2** [comment]
@galvesribeiro Download the zip from https://github.com/nunit/nunit-console/releases and run `nunit3-console "path\to\test\assembly.dll"` at the command line.

**Comment by jnm2** [comment]
@JustinRChou can we make these new-style .csproj's part of the VS adapter tests, both .NET Standard and .NET Framework? A single project can multitarget and produce both binaries.

**Comment by CharliePoole** [comment]
Issue moved to [nunit/nunit3-vs-adapter #297](https://github.com/nunit/nunit3-vs-adapter/issues/297) via [**ZenHub**](https://www.zenhub.com/)

**Comment by galvesribeiro** [comment]
@jnm2:

![image](https://cloud.githubusercontent.com/assets/4714040/21958310/97a29bd4-da92-11e6-8566-45c42f3f8e3e.png)

**Comment by CharliePoole** [comment]
Moved this to the nunit3 adapter repository. Further comments there on nunit/nunit3-vs-adapter#297

@jnm2 Have you been able to run .Net Standard projects under the console runner? We haven't announced support for that yet, but if it's working... :smile: it would be a nice surprise!

**Comment by jnm2** [comment]
> Have you been able to run .Net Standard projects under the console runner?

No- I can't reference the framework from a .NET Standard 1.4 test project or maybe it would :smile:

**Comment by CharliePoole** [comment]
Only asking because you told our user to do it.... 😮

**Comment by jnm2** [comment]
Except he's `netstandard16` so he actually can reference the framework, and I wondered if it was just a nunit3-vs-adapter shortcoming or if it was also a shortcoming with nunit-console.

**Comment by CharliePoole** [comment]
I guess it's possible that the console could "accidentally" work with netstandard16, but I kind of doubt it. Again... it's a feature we haven't introduced because we only introduced the netstandard build to the framework about 15 minutes ago in software development time. 😄 

The separation of the framework and engine is going to cause lots of short-term gaps like this. It's the tactical price we decided to pay for the strategic advantages we gain.
